input/postgres/schema.go

```diff
@@ -37,7 +37,7 @@ func CollectAllSchemas(server *state.Server, collectionOpts state.CollectionOpts
 		collected[dbName] = true
 		psNext, tsNext, databaseOid, err := collectOneSchema(server, collectionOpts, logger, ps, ts, dbName, ts.Version, systemType)
 		if err != nil {
-			logger.PrintVerbose("Failed to collect schema metadata for database %s: %s", dbName, err)
+			logger.PrintWarning("Failed to collect schema metadata for database %s: %s", dbName, err)
```

Comment:
I wonder if we should make this dependent on `collectionOpts.TestRun`? (i.e. only output it as a warning during the test run - otherwise this becomes very repetitive, e.g. in cases where we are missing a cloud provider internal DB from the `isCloudInternalDatabase` list)

Reply:
Yeah, this was banged out quickly on our support triage call with Sean, and we talked about that issue a bit--I mostly opened the PR as a straw man. We think this particular customer may have been hitting https://github.com/pganalyze/collector/pull/211 but it's hard to diagnose because this error is only logged in verbose mode. I think it's pretty unlikely anyone is actually running collection with verbose mode turned on (and instructing customers on how to turn it on is also complicated, especially given variations for the different package/Docker/Heroku deployment mechanisms and that it's not obvious how a customer has deployed the collector).

I think restricting it to test runs for now is a good start. But do customers re-test the collector after upgrading it? I'm not arguing for this change specifically, but I do think we should offer more visibility into collection issues to give customers confidence things are set up correctly (and clear diagnostics when they're not).